src/lib/divider/divider.scss

```diff
@@ -0,0 +1,66 @@
+.mat-divider {
+  display: block;
+  margin: 0;
+  border-top-width: 1px;
+  border-top-style: solid;
+
+  &.mat-divider-vertical {
+    border-top: 0;
+    border-right-width: 1px;
+    border-right-style: solid;
+    position: static;
+    width: auto;
+  }
+
+  &.mat-divider-inset {
+    margin-left: 80px;
+    [dir='rtl'] & {
+      margin-left: auto;
+      margin-right: 80px;
+    }
+  }
+}
+
+.mat-list-item {
+  .mat-divider {
+    position: absolute;
```

Comment:
There seems to be inconsistent spacing between attributes, can you make the spacing more consistent?

Reply:
Will fix